src/aiy/_drivers/_tts.py

```diff
@@ -20,6 +20,7 @@
 import subprocess
 import tempfile
 from aiy import i18n
+from aiy import audio
```

Comment:
Remove - this creates a circular import between the two modules, which could lead to confusing problems later.